mobile-app/app/api/persistence/portfolio_currency_storage.ts

```diff
@@ -0,0 +1,18 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { PortfolioButtonGroupTabKey } from '@screens/AppNavigator/screens/Balances/components/TotalPortfolio'
+
+const KEY = 'WALLET.PORTFOLIO_CURRENCY'
+
+async function set (denominationCurrency: NonNullable<PortfolioButtonGroupTabKey>): Promise<void> {
+    await AsyncStorage.setItem(KEY, JSON.stringify(denominationCurrency))
+}
+
+async function get (): Promise<PortfolioButtonGroupTabKey> {
+    const val = await AsyncStorage.getItem(KEY) ?? JSON.stringify('USDT')
+    return JSON.parse(val)
```

Comment:
Can we update this to 

const val = await AsyncStorage.getItem(key)
return val != null ? JSON.parse(val) : 'USDT'

So we only parse when it's from AsyncStorage. Otherwise, no need to stringify and parse USDT again